relayer-cli/src/commands/tx/packet.rs

```diff
@@ -37,7 +37,7 @@ impl Runnable for TxRawPacketRecvCmd {
             src_port_id: self.src_port_id.clone(),
             src_channel_id: self.src_channel_id.clone(),
         };
-        let mut link = match Link::new_from_opts(chains.src, chains.dst, opts) {
+        let mut link = match Link::new_from_opts(chains.src, chains.dst, opts, false) {
```

Comment:
why not use the config setting?

Reply:
This is the CLI for doing `tx raw packet-recv`. This means under the hood we're using the `SyncSender` which synchronously waits to confirm transactions. We are not using the `AsyncSender` here, so not sure having tx confirmations would be appropriate.

Reply:
right! missed that one. I hope we move all CLIs and workers to async at some point.